src/views/app/detail-panel/edit/edit-view.tsx

```diff
@@ -245,20 +245,39 @@ export const EditView = React.forwardRef<EditViewHandle, EditViewProp>(function
 		[createSnackbar, editorId]
 	);
 
-	const onSendError = useCallback((): void => {
-		createSnackbar({
-			key: `mail-${editorId}`,
-			replace: true,
-			severity: 'error',
-			label: t('label.error_try_again', 'Something went wrong, please try again'),
-			autoHideTimeout: TIMEOUTS.SNACKBAR_DEFAULT_TIMEOUT,
-			hideButton: true
-		});
-		createEditBoard({
-			action: EditViewActions.RESUME,
-			actionTargetId: editorId
-		});
-	}, [createSnackbar, editorId]);
+	function isErrorAboutInvalidRecipient(error?: unknown): boolean {
+		return typeof error === 'string' && error.includes('5.1.1');
+	}
+
+	const onSendError = useCallback(
+		(error?: unknown): void => {
+			let timeout = TIMEOUTS.SNACKBAR_DEFAULT_TIMEOUT;
+			let message = t('label.error_try_again', 'Something went wrong, please try again');
+			if (isErrorAboutInvalidRecipient(error)) {
+				const errorStr = typeof error === 'string' ? error : JSON.stringify(error);
+				const match = /\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,}\b/.exec(errorStr);
+				const invalidAddress = match?.[0] ?? 'unknown';
```

Comment:
I think we can avoid the fallback to 'unkown' since it will cause confusions:

 If no match is found the message string will look like:  _The recipient address "unknown" does not exist or is invalid_ . Now the _unkown_ is confusing to the end user since he never add any address which matches _unkown_. 

I propose to simply fallback to empty string, in this way the message will look like:

_The recipient address does not exist or is invalid_

Reply:
Maybe we can say "one or more addresses are not valid" if the address is unknown?